pages/common/rsync.md

```diff
@@ -11,6 +11,10 @@
 
 `rsync {{remote_host}}:{{path/to/remote_file}} {{path/to/local_directory}}`
 
+- Transfer any file or directory, in archive (to preserve attributes), resolving contained softlinks, and ignoring already transferred files (unless newer):
```

Comment:
I'll explain why the explanation for `azvhP` works and this can be improved.

From "in archive (to preserve attributes) and compressed (zipped) mode with verbose and human-readable progress", I can gather a is for archive, z is for zipped, v is for verbose, h is for human-readable and p is for progress.

Here I can make out archive, but nothing else.

There is another convention that we follow in tough situations like this. It is to wrap the letters with third brackets. For example: [a]rchive, [r]ecursively.

Is there a possibility we can change the description to use this convention and spell out each letter in "Laura" in the description ?

Reply:
done.  Let me know if you want to squash my commits. 

Also let me know if you think this pun is too silly to tldr.

Reply:
The letters should be part of the words themselves. I can do that myself. But what is the second `a` for ? I just see one for archive.